### PR TITLE
Drafts for Basic Upgrade Steps and Upgrade Overview

### PR DESCRIPTION
**Pull request by jhinkey**
Hi JR,

Here are my latest changes to these two articles. I **hope** the Basic Upgrade Steps aren't too long, but I found all the steps mentioned to be either essential (e.g., updating configs) or worthwhile (e.g. exporting all configs) to do. Y 

**Upgrade Overview**

1. Refer to "database upgrade". Auto-upgrade and the upgrade tool upgrade your database. I don't want to give the false impression that
doing the data upgrade is all you need to do or that when you're done the data upgrade, you're definitely done. The two main parts of upgrade
are database/data upgrade and code upgrade. Our new installations (and any new versions of MP apps) contain the upgraded code. The DXP
installation and MP apps contain the database upgrade processes that auto upgrade and the database upgrade tool execute. You could conceivably
complete a database upgrade run, but not have upgraded your MP apps, custom code, and DXP configs/properties. 

2. Added topic "Update Custom Plugin Code" since it's a part (a big part) of upgrading. 

3. Add updating MP apps to "Preparations and Planning", as it's something you start doing on the current DXP before upgrading. Of course, after you upgrade DXP, you can/should install the newest MP apps (I've added both step to the basic upgrade).

--- 

**Basic Upgrade Steps**

1. I updated the intro paragraphs to explain what's different (and good) about the basic upgrade steps and why they might apply to the reader.

2. Added common Preliminary Steps that should be done on the current server, before the data upgrade.
   1. Update the MP apps so important intermediate data upgrade steps aren't skipped. You could skip this, but your MP apps might not work, which would require you to circle back at some point. Similar to upgrading custom code--but MP apps are a quicker win IMO.
   2. Export the System Settings as you can do it with one button click for ALL the settings (setting you may have set manually in the UI).
   3. Back up your installation. (We've already mentioned this, but we might as well put it here if we're listing prelim steps.

_Docker Image_

1. For database setup, use your portal-ext.properties. We're already going to need the portal-ext.properties, so might as well use them for the database instead of 
messing around with the command line. [If you do "docker run" and mess up the env vars, the container is created and you have to delete it to
get rid of it--better to use portal properties when CREATING the container. After creating it, it's fine to override with command env vars].

2. For document library (file store), you're updating that info in the portal properties, as part of the config/property migration/upgrade.

3. Migrate/update the configs and portal properties before starting DXP and the database upgrade as you need the proper configuration for
DXP to work right. If you have to do it anyway, might as well do it where it's going to served best--starting the new DXP. 

4. The bind/volume mount in the docker run command is there so that you can modify configs, properties, and deploy apps. Bind mounts
need to be done when creating the container, which you only do once via "docker run...".

5. Mention Gogo shell for resolving any upgrade issues. 

6. Re-enable search indexer.

7. Disable auto-run as you only want to run it once. Use Gogo shell after that.

### CODE REVIEW

File: docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay-dxp/upgrade-basics/introduction-to-upgrading-liferay-dxp.md [hunk 1]
```diff
@@ -31,49 +30,69 @@ If your path includes upgrading to Liferay Portal 6.2, follow the [Liferay Porta
...
 
-Finally, review [Post-Upgrade Considerations](./post-upgrade-considerations.md) to ensure configurations and content are correct for your upgraded Liferay DXP installation. Your upgrade is now complete.
+Once you've completed the tasks outlined above, your upgrade is complete. 
+
+Review the [Post-Upgrade Considerations](./post-upgrade-considerations.md) to ensure configurations and content are correct for your upgraded Liferay DXP installation.
+
+If you are upgrading from an older version (7.1 and below) you may also need to [install Elasticsearch](../configuration-and-infrastructure/dxp-and-elasticsearch.md) to handle search indexing.
 
-## Related Information
+To get started,
```
Comment (jhinkey) [inline docs/dxp/7.x/en/installation-and-upgrades/upgrading-liferay-dxp/upgrade-basics/introduction-to-upgrading-liferay-dxp.md]: Sorry, I forgot to finish this transition.

**Comment by jhinkey** [comment]
I polished up the Overview... more.

**Comment by jhinkey** [comment]
@jrhoun I updated "Using the Database Upgrade Tool". So now the overview, basic upgrade, and upgrade tool article drafts are ready! The other articles (which I'm updating) support them.

**Comment by jhinkey** [comment]
More to eyeball

**Comment by jhinkey** [comment]
updated **Post Upgrade Considerations** and **Migrating Configurations and Properties**

**Comment by jrhoun** [comment]
Just starting to review.

**Comment by jrhoun** [comment]
See https://github.com/jhinkey/liferay-learn/pull/8

Thanks @jhinkey !

**Pull request closed without merging**